dags/vz_refresh_location_crashes.py

```diff
@@ -0,0 +1,73 @@
+"""
+This DAG refreshes the location_crashes_view in the Vision Zero database
+"""
+
+import os
+from pendulum import datetime, duration
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.onepassword import get_env_vars_task
+from utils.slack_operator import task_fail_slack_alert
+
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT")
+secrets_env_prefix = None
+
+if DEPLOYMENT_ENVIRONMENT == "production":
+    secrets_env_prefix = "prod"
+elif DEPLOYMENT_ENVIRONMENT == "staging":
+    secrets_env_prefix = "staging"
+else:
+    secrets_env_prefix = "dev"
```

Comment:
my local airflow's env file has the deployment_environment set as staging, so I ran this a few times and had it fail since it was not using the dev secrets. 

that failure made me notice that the staging endpoint in 1password has a typo